File: Backend/src/main/java/com/feedle/feedleapi/Networking/AddCommentRequest.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.Comment;

public class AddCommentRequest extends Request {
    private Comment comment;
    public AddCommentRequest(Comment comment)
    {
        super(RequestType.AddComment);
        this.comment = comment;
    }

    public Comment getComment() {
        return comment;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/AddCommentRequest.cs
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class AddCommentRequest : Request
    {
        [JsonPropertyName("comment")]
        public Comment Comment { get; set; }
        public AddCommentRequest(Comment comment) : base(RequestType.AddComment)
        {
            this.Comment = comment;
        }
    }
}